### DISCUSSION TITLE
Magma AGW deployment over ubuntu VM/Container other way than vagrant or bare-metal

### DISCUSSION THREAD

**Discussion by logic2code** [magma/magma#6021] [General]
Hi all,

is there  some experiences or procedures to deploy Magma AGW components at Ubuntu VM or in containers ?

best regards,
Jatinder

**Comment by electronjoe** [comment]
Hello @Jatinder01!

There is ongoing work to containerize the Magma AGW components over the next release cycle (~quarter).

There may be increasing traffic on slack within [#project-agw-containerization](https://magmacore.slack.com/archives/C01BZT3SEAV).
